Resources/Prototypes/_NF/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```diff
@@ -49,3 +49,15 @@
     contents:
       - id: BoxSurvival
       - id: RubberStampStc
+
+- type: entity
+  noSpawn: true
+  parent: ClothingBackpackPilot
+  id: ClothingBackpackPilotFilled
+  components:
+    - type: StorageFill
+      contents:
+      - id: BoxSurvival
+      - id: Lighter
+      - id: CigPackMixed
+      - id: DoubleEmergencyOxygenTank
```

Comment:
CigPackMixed and DoubleEmergencyOxygenTank?
A bit too much
Extended one and normal cig pack please

Reply:
also done :(